apps/cli/src/commands/manually-challenge-assertion.ts

```diff
@@ -43,7 +43,9 @@ export function manuallyChallengeAssertion(cli: Vorpal) {
             await submitAssertionToReferee(
                 secretKey,
                 assertionId,
-                assertionNode,
+                assertionId - 1,
+                assertionNode.confirmData,
+                assertionNode.createdAtBlock,
```

Comment:
There is more to be considered here, this command is used to manually submit a challange, should this not also batch if multiple assertions happened in between ?